### PR TITLE
Reminder

### PR DESCRIPTION
**Pull request by bdt0-rocklee**
A small plugin that is used to set reminders

### CODE REVIEW

File: plugins/reminder.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		
+		def usage
+			"!remind <count> <unit> 'Message'."
+		end
+		
+		def match(cmd)
+			cmd =~ /^(!)?remind$/
+		end
+	
+	match /remind (\d+) (\w+) (.+)/, method: :set_reminder
```
Comment (woodruffw) [inline plugins/reminder.rb]: fix the indentation here

**Comment by bdt0-rocklee** [comment]
changed the file info

File: plugins/reminder.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		
+		def convert_to_sec(count, unit)
+			count = (count*60 ) if unit =~ /min/
+			count = (count*3600) if unit =~ /hour/
+			return count
+		end
+		
+		def set_reminder(m, count, unit, message)
+			  count = convert_to_sec(count, unit) if unit =~ /min|hour/
```
Comment (woodruffw) [inline plugins/reminder.rb]: instead of putting the logic in `convert_to_sec`, just use a conditional block here

File: plugins/reminder.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#  -*- coding: utf-8 -*-
+#  reminder.rb
+#  Author: slackR | slackErEhth77
+#  ------------------------
+#  A Cinch plugin to set reminder  for a given time.
```
Comment (woodruffw) [inline plugins/reminder.rb]: minor formatting here

File: plugins/reminder.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+#  ------------------------
+#  This code is licensed by slackErEhth77 under the MIT License.
+#  http://opensource.org/licenses/MIT
+
+require_relative 'yossarian_plugin'
+
+	class Reminder < YossarianPlugin
+		include Cinch::Plugin
+
```
Comment (woodruffw) [inline plugins/reminder.rb]: add `use_blacklist` below this include so that the plugin responds to blacklist blocks

File: plugins/reminder.rb [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+#  This code is licensed by slackErEhth77 under the MIT License.
+#  http://opensource.org/licenses/MIT
+
+require_relative 'yossarian_plugin'
+
+	class Reminder < YossarianPlugin
+		include Cinch::Plugin
+		
+		def usage
+			"!remind <count> <unit> 'Message'."
```
Comment (woodruffw) [inline plugins/reminder.rb]: use single quotes here, instead of 'Message' do <message>. add a short description like other plugins

**Comment by woodruffw** [comment]
Fix the indentation, please. It should be one tab per scope, non-spaces.

File: plugins/reminder.rb [hunk 6]
```diff
@@ -0,0 +1,40 @@
+#  -*- coding: utf-8 -*-
+#  reminder.rb
+#  Author: slackR | slackErEhth77
+#  ------------------------
+#  A Cinch plugin to set reminder  for a given time.
+#  ------------------------
+#  This code is licensed by slackErEhth77 under the MIT License.
+#  http://opensource.org/licenses/MIT
+
+ require_relative 'yossarian_plugin'
```
Comment (woodruffw) [inline plugins/reminder.rb]: remove leading space

File: plugins/reminder.rb [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+		def match(cmd)
+			cmd =~ /^(!)?remind$/
+		end
+	
+	match /remind (\d+) (\w+) (.+)/, method: :set_reminder
+		
+	def convert_to_sec(count, unit)
+	
+	    if unit =~ /min/ 
+	      count = (count*60 )
```
Comment (woodruffw) [inline plugins/reminder.rb]: use tabs, not spaces

File: plugins/reminder.rb [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+	    else
+	      count = (count*3600)
```
Comment (woodruffw) [inline plugins/reminder.rb]: use tabs, not spaces

**Pull request merged**